.travis.yml

```diff
@@ -5,32 +5,18 @@ git:
 
 jobs:
   include:
-    - name: Vim 7.4
-      env:
-      - VIM_VERSION=v7.4
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-v7.4.txt
-    - name: Vim v8
-      env:
-      - VIM_VERSION=v8.0.0000
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-v8.0.txt
-    - name: Vim master
-      env:
-      - VIM_VERSION=master
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-master.txt
     - name: Installed Vim with checks
       env:
       - VIM_VERSION=installed
-      - MAKE_TARGET="clean_compiled check checkpy js/test py/test test/node_position/test_position.out"
+      - MAKE_TARGET="checkvim"
```

Comment:
why did you remove `clean_compiled check checkpy js/test py/test test/node_position/test_position.out` ?

Reply:
It was also temporary to get faster feedback on CI.. :)